src/model/queries/update_user_challenge.js

```diff
@@ -0,0 +1,14 @@
+const db = require('../database/db_connection');
+
+const updateUserChallenge = (id, status) =>
+  db.query(
+    `
+  UPDATE user_challenges
+  SET status = $2
+  WHERE id= $1
```

Comment:
I guess because this is a template string ESLind doesn't tidy it up but we could clean up and formalise our DB queries...